QModManager/API/SMLHelper/Patchers/OptionsPanelPatcher.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace QModManager.API.SMLHelper.Patchers
+{
+    using Harmony;
+    using Options;
+    using System.Collections.Generic;
+    using UnityEngine;
+    using UnityEngine.Events;
+    using UnityEngine.UI;
+
+    internal static class OptionsPanelPatcher
+    {
+        internal static SortedList<string, ModOptions> modOptions = new SortedList<string, ModOptions>();
+
+        internal static void Patch(HarmonyInstance harmony)
+        {
+            harmony.Patch(AccessTools.Method(typeof(uGUI_OptionsPanel), "AddTabs"),
+                postfix: new HarmonyMethod(AccessTools.Method(typeof(OptionsPanelPatcher), "AddTabs_Postfix")));
+        }
+
+        internal static void AddTabs_Postfix(uGUI_OptionsPanel __instance)
+        {
+            uGUI_OptionsPanel optionsPanel = __instance;
+            
+            // Start the modsTab index at a value of -1
+            var modsTab = -1;
+            // Loop through all of the tabs
+            for (int i = 0; i < optionsPanel.tabsContainer.childCount; i++)
+            {
+                // Check if they are named "Mods"
+                var text = optionsPanel.tabsContainer.GetChild(i).GetComponentInChildren<Text>(true);
+                if (text != null && text.text == "Mods")
+                {
+                    // Set the tab index to the found one and break
+                    modsTab = i;
+                    break;
+                }
+            }
+            // If no tab was found, create one
+            if (modsTab == -1)
+            {
+                modsTab = optionsPanel.AddTab("Mods");
+            }
+
+            // Maybe this could be split into its own file to handle smlhelper options, or maybe it could be removed alltogether
+            optionsPanel.AddHeading(modsTab, "SMLHelper");
```

Comment:
merge with qmodmanager?

Reply:
Yes. Merge items from the Mod Options menu for SMLHelper and QMM together.